modules/ROOT/pages/B2B-overview.adoc

```diff
@@ -3,12 +3,13 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Partner Manager enables you to manage transmissions of business messages with a large number of trading partners through the creation and deployment of message flows. 
-A transmission is a single communication event during which a payload is transferred between two business entities. 
-These payloads can contain one or more business messages. 
+Anypoint Partner Manager enables you to manage transmissions of business messages with a large number of trading partners through the creation and deployment of message flows.
+A transmission is a single communication event during which a payload is transferred between two business entities.
+These payloads can contain one or more business messages.
 A message flow is the path along which a message travels from its source, through a DataWeave transformation map, to its target.
 
-You'll need to perform some tasks before you can begin transmitting messages with your trading partners. In Anypoint Partner Manager, you set yourself up as a host partner, configure your trading partners, and then create message flows between you and your trading partners.
+You'll need to perform some tasks before you can begin transmitting messages with your trading partners.
+In Anypoint Partner Manager, you set yourself up as a host partner, configure your trading partners, and then create message flows between you and your trading partners.
```

Comment:
This text confuses me a bit, in terms of organization (again, check your TOC).
If this is an introduction to the deliverable, then this text ought to summarize the deliverable content, as indicated by all the Level-1 headings in it.
If this is a Level-1 topic, then it should summarize the Level-2 topics within it. 
However, there is just one Level-2 topic here, and it should be promoted as one of the first tasks you have to perform. "Preparing" is not a subtopic to "Using."